source/localizable/tutorial/routes-and-templates.md

```diff
@@ -7,6 +7,8 @@ Let's take a look at the application from the user perspective before we get sta
 We arrive at the home page which shows a list of rentals.
 From here, we will be able to navigate to an about page and a contact page.
 
+Keep in mind that generating the routes will break the main route until its added back in index route section, and means that navigating to [`http://localhost:4200/`](http://localhost:4200/) yields an loading an empty white page. 
```

Comment:
Instead of saying adding a route breaks the main route, you can explain that adding your first route removes the Ember welcome screen.

```
Keep in mind that generating your first route will remove the default Ember welcome page.  
Now when you navigate to [`http://localhost:4200/`](http://localhost:4200/), the app will load an empty white page.  
We will fix this behavior later on in this section.
```